pyne/utils.py

```diff
@@ -52,12 +52,21 @@ class QAWarning(UserWarning):
                   'y': 86400.0*365.25,
                   'year': 86400.0*365.25,
                   'years': 86400.0*365.25,
-                  'ev': 1e-9 * 7.6e-8 / 6.03,
-                  'kev': 1e-12 * 7.6e-8 / 6.03,
-                  'mev': 1e-15 * 7.6e-8 / 6.03,
                   }
 
 
+# Energy to half-life conversion:  T1/2= ln(2) × (h/2 pi) / energy
+# See http://www.nndc.bnl.gov/nudat2/help/glossary.jsp#halflife
+# NIST CODATA https://physics.nist.gov/cgi-bin/cuu/Value?hbar
+#    h-bar = 1.054 571 800(13) x 1e-34 J 
+#    1 J = 6.241 509 126(38) x 1e18 eV
+HBAR_LN2 = 4.5623775832376968e-16  # h-bar ln(2) in eV s
+energy_conv_dict = {'ev': HBAR_LN2,
+                    'kev': 1e-3 * HBAR_LN2,
```

Comment:
I don't know that this is quite correct.  I don't think that this should have the `ln(2)` per se.  That is only relevant for converting half-lives, but there are other things measured in MeV that could be converted to time.